backend/src/main/java/gov/cdc/usds/simplereport/service/DataHubUploaderService.java

```diff
@@ -193,15 +193,13 @@ private void uploadCSVDocument(String apiKey) throws RestClientException {
     _resultJson = restTemplate.postForObject(url, contentsAsResource, String.class);
   }
 
-  @AuthorizationConfiguration.RequirePermissionExportTestEvent
-  public String createTestCSVForDataHub(String lastEndCreateOn) {
+  @AuthorizationConfiguration.RequireGlobalAdminUser
```

Comment:
Does this mean this permission is no longer needed? Or are we going to add a site-specific export?

Reply:
This export is not site-specifc, so the permission required to invoke this method needed to be global rather than organizational.

Reply:
@benwarfield-usds: If we want to add site-specific-exporting as a feature, I think the existing code needs a bunch of work to stream the results. Right now the function is only use is for site-admins to export test results in case of datastream upload failures.

Reply:
So in that case should we be deleting the permission and related code?

Reply:
yes, but an immediate mitigation had to go out in a hurry.

Reply:
Not trying to be argumentative here, just want to know if we can delete some stuff. 🙃